src/Header.test.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
```

Comment:
I don't think this will fail if I change 
`updateAuthenticationInfo={(authResponse) => props.updateAuthenticationInfo(authResponse)}`

to 

`updateAuthenticationInfo={(authResponse) => props._updateAuthenticationInfo(authResponse)}`

Test and if this doesn't fail, we need to add tests.

Reply:
Added tests.